WordPress/gutenberg
# Data views: Reduce page selector prominence

## What?
Reduce the visual prominence of the page selector in the pagination row.

## Why?
It currently garners too much attention.

## How?
Unfortunately, without building this into the component (https://github.com/WordPress/gutenberg/issues/57720) I had to overwrite some of the included styles. I don't feel great about shipping this, but at the same time I'd like to update the visuals here. cc @jasmussen @ciampo @mirka for thoughts on that.

## Testing Instructions
1. Open one of the management views (pages, templates, patterns) in the site editor.
2. Observe the updated page selector design.

### Before
<img width="454" alt="Screenshot 2024-01-24 at 14 33 18" src="https://github.com/WordPress/gutenberg/assets/846565/3961415c-7b7e-418d-8721-0c601283876a">


### After
<img width="355" alt="Screenshot 2024-01-24 at 14 32 11" src="https://github.com/WordPress/gutenberg/assets/846565/aaab9703-89b7-419a-8201-1e68ba0909f3">
